tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2001 @@
+            cls.sc_obj.delete(resource_name=cls.retain_nfs_sc_name)
+            log.info(f"Wait until the SC, {cls.retain_nfs_sc_name} is deleted.")
+            cls.sc_obj.wait_for_delete(resource_name=cls.retain_nfs_sc_name)
+
+        # Check if NFS client connection was established and clean up mount
+        if (
+            hasattr(cls, "_TestNfsExport__nfs_client_connection")
+            and cls._TestNfsExport__nfs_client_connection
+        ):
+            try:
+                con = cls._TestNfsExport__nfs_client_connection
+                retcode, stdout, _ = con.exec_cmd("findmnt -t nfs4 " + cls.test_folder)
```

Ruff (0.15.21)

[warning] Unpacked variable `retcode` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)